lib/jobs/updateTemplateRepos.nix

```diff
@@ -0,0 +1 @@
+{ rev, pkgs, repos }: {}
```

Comment:
What is this file supposed to eventually do? I'm just not sure why lib needs to update template repos...

Reply:
It's `./lib/jobs`. It's kind of similar to `.github/workflows`. Usually, pure chore.